packages/as-types/assembly/argument.ts

```diff
@@ -732,9 +732,11 @@ export class Args {
    * "args doesn't know how to serialize the given type."
    *
    * @param arg - the argument to add
+   * @typeParam T - The type of the object to deserialize
+   * @typeParam U - The type of the object to instantiate if the object is an array of serializable objects
```

Comment:
How would these two be different anyway ?

Reply:
They are when used with array. U is needed only when working on array of serializable.
For example:
```typescript
new Args().add<Array<Divinity>, Divinity>(emptyArray).serialize();
```